tests/test_other.py

```diff
@@ -9966,13 +9966,18 @@ def test_assertions_on_ready_promise(self):
     # the breaking change in #10697 Module() now returns a promise, and to get
     # the instance you must use .then() to get a callback with the instance.
     create_test_file('test.js', r'''
+      // Adds a noop catch handler to avoid unhandled promise rejection error for the process
+      function addCatch(promise) {
+        promise.catch(() => {});
+        return promise;
+      }
       try {
-        Module()._main;
+        addCatch(Module())._main;
```

Comment:
Does this still test for the case we are interesting it?  i.e. when the user tries to using the promise as a module?